[Feature] - Since 2.0.0 Wildfy Server refuses to start

**Is your feature request related to a problem? Please describe.**
Since version 2.0.0 Ocelot Agent prevents Wildfly servers from starting. `rocks.inspectit.ocelot.agent.AgentMain` initializes the logging system - but Wildfly is very stubborn at this point and expects `org.jboss.logmanager.LogManager` as LogManager.
This is a common problem when using Java agents with Wildfly.

`Caused by: java.lang.IllegalStateException: WFLYLOG0078: The logging subsystem requires the log manager to be org.jboss.logmanager.LogManager. The subsystem has not be initialized and cannot be used. To use JBoss Log Manager you must add the system property "java.util.logging.manager" and set it to "org.jboss.logmanager.LogManager"
at org.jboss.as.logging.LoggingExtension.initialize(LoggingExtension.java:195)
at org.jboss.as.controller.extension.ExtensionAddHandler.initializeExtension(ExtensionAddHandler.java:131)
at org.jboss.as.controller.extension.ExtensionAddHandler.initializeExtension(ExtensionAddHandler.java:103)
at org.jboss.as.controller.extension.ParallelExtensionAddHandler$ExtensionInitializeTask.call(ParallelExtensionAddHandler.java:144)
at org.jboss.as.controller.extension.ParallelExtensionAddHandler$ExtensionInitializeTask.call(ParallelExtensionAddHandler.java:127)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at org.jboss.threads.ContextClassLoaderSavingRunnable.run(ContextClassLoaderSavingRunnable.java:35)
at org.jboss.threads.EnhancedQueueExecutor.safeRun(EnhancedQueueExecutor.java:1982)
at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.doRunTask(EnhancedQueueExecutor.java:1486)
at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1377)
at java.lang.Thread.run(Thread.java:748)
at org.jboss.threads.JBossThread.run(JBossThread.java:508)`

**Describe the solution you'd like**
Update documentation how to configure Wildfly properly.

**Java 8**
```
-Djava.util.logging.manager=org.jboss.logmanager.LogManager
-Djboss.modules.system.pkgs=org.jboss.byteman,org.jboss.logmanager  
-Xbootclasspath/p:<WILDFLY_HOME>/modules/system/layers/base/org/jboss/logmanager/main/jboss-logmanager-<version>.jar  
-Xbootclasspath/p::<WILDFLY_HOME>/modules/system/layers/base/org/wildfly/common/main/wildfly-common-<version>.jar
-javaagent:inspectit-ocelot-agent.jar
```

**Java 11**
```
-Dsun.util.logging.disableCallerCheck=true
-Djava.util.logging.manager=org.jboss.logmanager.LogManager
-Djboss.modules.system.pkgs=org.jboss.byteman,org.jboss.logmanager
-Xbootclasspath/p:<WILDFLY_HOME>/modules/system/layers/base/org/jboss/logmanager/main/jboss-logmanager-<version>.jar
-Xbootclasspath/p::<WILDFLY_HOME>/modules/system/layers/base/org/wildfly/common/main/wildfly-common-<version>.jar
-javaagent:inspectit-ocelot-agent.jar
```

--- a/inspectit-ocelot-agent/src/main/java/rocks/inspectit/ocelot/agent/AgentMain.java
+++ b/inspectit-ocelot-agent/src/main/java/rocks/inspectit/ocelot/agent/AgentMain.java
@@ -16,7 +16,6 @@ import java.security.AllPermission;
 import java.security.CodeSource;
 import java.security.PermissionCollection;
 import java.util.jar.JarFile;
-import java.util.logging.Logger;
 
 /**
  * Entry point of the agent.
@@ -45,8 +44,6 @@ public class AgentMain {
 
     private static final String PUBLISH_OPEN_TELEMETRY_TO_BOOTSTRAP_PROPERTY = "inspectit.publishOpenTelemetryToBootstrap";
 
-    private static Logger LOGGER = Logger.getLogger(AgentMain.class.getName());
-
     /**
      * Main method for attaching the agent itself to a running JVM.
      *
@@ -73,7 +70,7 @@ public class AgentMain {
         boolean loadOpenTelemetryJarToBootstrap = null != System.getProperty(PUBLISH_OPEN_CENSUS_TO_BOOTSTRAP_PROPERTY) ? "true".equalsIgnoreCase(System.getProperty(PUBLISH_OPEN_CENSUS_TO_BOOTSTRAP_PROPERTY)) : "true".equalsIgnoreCase(System.getProperty(PUBLISH_OPEN_TELEMETRY_TO_BOOTSTRAP_PROPERTY));
         // check for deprecated JVM property
         if (null != System.getProperty(PUBLISH_OPEN_CENSUS_TO_BOOTSTRAP_PROPERTY)) {
-            LOGGER.warning("You are using the deprecated JVM property '" + PUBLISH_OPEN_CENSUS_TO_BOOTSTRAP_PROPERTY + "'. Please use the new JVM property '" + PUBLISH_OPEN_TELEMETRY_TO_BOOTSTRAP_PROPERTY + "'. inspectIT Ocelot has moved from OpenCensus to OpenTelemetry. However, applications using the OpenCensusAPI are still supported through the opentelemetry-opencensus-shim ");
+            System.err.println("You are using the deprecated JVM property '" + PUBLISH_OPEN_CENSUS_TO_BOOTSTRAP_PROPERTY + "'. Please use the new JVM property '" + PUBLISH_OPEN_TELEMETRY_TO_BOOTSTRAP_PROPERTY + "'. inspectIT Ocelot has moved from OpenCensus to OpenTelemetry. However, applications using the OpenCensusAPI are still supported through the opentelemetry-opencensus-shim ");
         }
 
         try {

--- a/inspectit-ocelot-core/src/main/java/rocks/inspectit/ocelot/core/config/propertysources/EnvironmentInformationPropertySource.java
+++ b/inspectit-ocelot-core/src/main/java/rocks/inspectit/ocelot/core/config/propertysources/EnvironmentInformationPropertySource.java
@@ -44,7 +44,7 @@ public class EnvironmentInformationPropertySource extends PropertiesPropertySour
         URL agentJar = Instances.AGENT_JAR_URL;
         if (agentJar != null) {
             try {
-                return new File(agentJar.toURI()).getParent();
+                return new File(agentJar.getPath()).getParent();
             } catch (Exception e) {
                 log.error("Error detecting JAR directory {}", e);
             }

